File: rankingEducacional/src/java/model/Aluno.java
package model;

public class AlunoFator {

  private String matriculaAluno;
  private int serieEmCurso;
  private int anoLetivo;
  private String observacao;

  public Alunor() {}

  public String getMatriculaAluno() {
    return matriculaAluno;
  }

  public void setMatriculaAluno(String matriculaAluno) {
    this.matriculaAluno = matriculaAluno;
  }

  public int getSerieEmCurso() {
    return serieEmCurso;
  }

  public void setSerieEmCurso(int serieEmCurso) {
    this.serieEmCurso = serieEmCurso;
  }

  public int getAnoLetivo() {
    return anoLetivo;
  }

  public void setAnoLetivo(int anoLetivo) {
    this.anoLetivo = anoLetivo;
  }

  public String getObservacao() {
    return observacao;
  }

  public void setObservacao(String observacao) {
    this.observacao = observacao;
  }
}

File: App/models/aluno.php
<?php

class Aluno
{
    private  $matriculaAluno;
    private  $serieEmCurso;
    private  $anoLetivo;
    private  $observacao;

    function __construct()
    {
    }

    public function getMatriculaAluno()
    {
        return $this->matriculaAluno;
    }

    public function setMatriculaAluno($matriculaAluno)
    {
        $this->matriculaAluno = $matriculaAluno;
    }

    public function getSerieEmCurso()
    {
        return $this->serieEmCurso;
    }

    public function setSerieEmCurso($serieEmCurso)
    {
        $this->serieEmCurso = $serieEmCurso;
    }

    public function getAnoLetivo()
    {
        return $this->anoLetivo;
    }

    public function setAnoLetivo($anoLetivo)
    {
        $this->anoLetivo = $anoLetivo;
    }

    public function getObservacao()
    {
        return $this->observacao;
    }

    public function setObservacao($observacao)
    {
        $this->observacao = $observacao;
    }
}
